examples/time/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "uniffi-example-time"
```

Comment:
In the early days of this project, the only way we had of running tests was to create example crates, so we added things in our examples that didn't really make narrative sense, but which exercised important parts of the code. The "rondpoint" example is a good example of this - anyone trying to actually learn UniFFI by example using that crate will probably have a bad time.

These days, we have a separate `fixtures` directory for things that are designed to exercise the code itself, rather than to provide examples of how to use the code, and I think it helps keep the repo a bit easier to navigate around in.

How would you feel about moving this `uniffi-example-time` crate into the `./fixtures` directory?  I think you should be able to keep it mostly intact, perhaps just renaming it to `uniffi-fixture-time` or similar.